feature/bgp/afi_safi/oc_defaults/README.md

```diff
@@ -0,0 +1,68 @@
+  - [Test case-4] IPv4-unicast and IPv6-Unicast AFI-SAFIs enabled at Global level:
+    - Configuration at the neighbor level is same as in [Test case-1] except for IPv4-unicast and IPv6-unicast being enabled at the global level
+    - No configuration should be made at the peer-group AFI-SAFI level
+    - verification:
+      - For IPv4 neighbor, ensure that the IPv4 neighborship is up and both IPv4-unicast and IPv6-unicast capabilities are negotiated.
+      - For IPv6 neighbor ensure that the IPv6 neighborship is up and both IPv4-unicast and IPv6-unicast capabilities are negotiated.
+
+Config Parameter coverage:
+
+  /network-instances/network-instance/protocols/protocol/bgp/global/config/as
+  /network-instances/network-instance/protocols/protocol/bgp/global/config/router-id
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/config/auth-password
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/config/neighbor-address
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/config/peer-as
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/neighbor-address
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/config/enabled
+
+
+  /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/config/auth-password
+  /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/config/neighbor-address
+  /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/config/peer-as
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/config/peer-group/peer-group-name
+  /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/config/enabled
+
+  /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/config/enabled
+
+Telemetry Parameter coverage:
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/session-state
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/supported-capabilities
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/peer-type
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/peer-as
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/supported-capabilities
+
+  /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/state/peer-type
+  /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/state/peer-as
+  /network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/state/local-as
+  /network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/peer-group
```

Comment:
Should create a feature.textproto for these paths, but it can be in a future PR.